library/func_misc.py

```diff
@@ -75,3 +76,80 @@ def Call(self, pos_args, named_args):
 
         # Equivalent to no return value?
         return value.Null
+
+
+class StartsWith(vm._Callable):
+
+    def __init__(self):
+        # type: () -> None
+        """Empty constructor for mycpp."""
+        pass
+
+    def Call(self, pos_args, named_args):
+        # type: (List[value_t], Dict[str, value_t]) -> value_t
+
+        spec = ArgsSpec([value_e.Str, value_e.Str], {})
+        spec.AssertArgs("startswith", pos_args, named_args)
+
+        string = cast(value.Str, pos_args[0]).s
+        match = cast(value.Str, pos_args[1]).s
+
+        res = string.startswith(match)
+        return value.Bool(res)
+
+
+class Strip(vm._Callable):
+
+    def __init__(self):
+        # type: () -> None
+        """Empty constructor for mycpp."""
+        pass
+
+    def Call(self, pos_args, named_args):
+        # type: (List[value_t], Dict[str, value_t]) -> value_t
+
+        spec = ArgsSpec([value_e.Str], {})
```

Comment:
Now I see `{}` and see GC pressure in the common case :)    i.e. we could use None instead, even though that sorta uglifies the code

Reply:
We can put this off for later, but I have a feeling we'll do it eventually

Reply:
That's a good point. I'll leave this for now; the `ArgSpec` utility will likely have to be updated later as it cannot, for example, express method signatures for variadic functions.